apps/sms/desktop-mock/navigator_moz_settings_shim.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+/* ***********************************************************
+
+  Workaround to force the mozSettings in shim context use window one.
+  Need to be removed once all the event listeners are ready for shim.
+
+*********************************************************** */
+(function(window) {
+ 
+if (!parent.navigator.mozSettings) {
+  console.error('mozSettings for main window context is not ready!');
+  return;
+}
+
+navigator.mozSettings = parent.navigator.mozSettings;
+
+}(window));
```

Comment:
__nit__ new line at EOF and please remove ```_shim``` part from the name of the file for the consistency sake :)